csharp/extractor/Semmle.Extraction.CSharp.Standalone/CsProjFile.cs

```diff
@@ -71,6 +84,30 @@ public void ReadProjectFileAsXml(FileInfo filename)
             var projDir = filename.Directory;
             var root = projFile.DocumentElement;
 
+            // Figure out if it's dotnet core
+
+            bool netCoreProjectFile = root.GetAttribute("Sdk") == "Microsoft.NET.Sdk";
+
+            if(netCoreProjectFile)
+            {
+                var relativeCsIncludes2 =
```

Comment:
why the `2`?

Reply:
Fixed.

Reply:
There was a name clash but I've removed it by using `else` instead.